package-lock.json

```diff
@@ -13079,15 +13098,20 @@
             "integrity": "sha512-cliV19VHLJqFUYbz/XeWXe5CO6guzwd0yrrqqp0bmjlMP3ZZULY7fu8RTC4+3lmHwo6ESVDHFDsvjB15hcR5IA==",
             "dev": true
         },
+        "vuex": {
+            "version": "3.1.1",
+            "resolved": "https://registry.npmjs.org/vuex/-/vuex-3.1.1.tgz",
+            "integrity": "sha512-ER5moSbLZuNSMBFnEBVGhQ1uCBNJslH9W/Dw2W7GZN23UQA69uapP5GTT9Vm8Trc0PzBSVt6LzF3hGjmv41xcg=="
+        },
```

Comment:
Only keep this `vuex` part and revert all other change of this file. Or CSS incompatibility issue might be happen.